pkg/bastion/dbinit.go

```diff
@@ -661,15 +660,15 @@ func DBInit(db *gorm.DB) error {
 		ErrMsg: "sshportal was halted while the connection was still active",
 	}).Error
 }
-
+/* Implemented hard delete via Unscoped() method in Gorm library
```

Comment:
Can you completely delete the old implementation?

Reply:
Removed in 6acdc3c